render_data_doc.R

```diff
@@ -0,0 +1,17 @@
+# make this a package function? how to render from inst folder?
```

Comment:
if the template file is in `/inst`, it is elevated to the "root" folder of the package when it is installed.  So, assuming anyone running the function already has the package installed, you can refer to the Rmd file with:

```r
fs::path(fs::path_package("CODECtools"), "data_doc_template.Rmd")
```